snapcraft/internal/errors.py

```diff
@@ -175,6 +175,32 @@ def __init__(self, command, app_name):
         super().__init__(command=command, app_name=app_name)
 
 
+class InvalidAppCommandNotFound(SnapcraftError):
+
+    fmt = (
+        "Failed to generate snap metadata: "
+        "The specified command {command!r} defined in the app {app_name!r} does "
+        "not exist.\n"
+        "Ensure that {command!r} is relative to the prime directory."
```

Comment:
From a user perspective, I'm not sure what the 'prime' directory implies... especially as the building with lxd/multipass really hides the prime directory from the user (is there a better way than `snapcraft stage --shell; ls prime/`?)

Reply:
`snapcraft prime --shell-after` is better until we have introspection,
`find prime` will also be more appropriate

Reply:
Perhaps we can suggest that to the user?  Or state along the lines of "Ensure that {command!r} is installed with the correct path."

Reply:
@clobrano hi, I just noticed the error message doesn't follow the pattern we have across the board, can you change the `: ` to `.\n`

we are planning a general change for error messages, but let's not go with to patterns at once.
Applying @cjp256's suggestion would be nice too.

Reply:
Ah, I didn't notice the pattern. Sure I'll do both changes

Reply:
Sorry for the delay, shall I convert all the occurences of `:` in `.\n`? Or maybe only in the error messages I changed? I see that this pattern is not observed in this file.

Reply:
@clobrano You are right, your pattern seems consistent. Just swap `"Ensure that {command!r} is relative to the prime directory."` with `"Ensure that {command!r} is installed with the correct path."`